lua/lazyvim/plugins/extras/lang/gleam.lua

```diff
@@ -0,0 +1,17 @@
+return {
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "gleam" })
+      end
+    end,
+  },
+
+  {
+    "neovim/nvim-lspconfig",
+    init = function()
+      require("lspconfig").gleam.setup({})
+    end,
```

Comment:
```suggestion
	opts = {
    	servers = {
    		gleam = {},
    	},
    },
```
This way the server will also get auto-installed by Mason.

Reply:
Oh it seems Mason has deprecated the installation for Gleam and it should be installed another way. Still, I believe the above changes make more sense in the spirit of how LazyVim configures LSP servers.

Reply:
True this looks sensible

Reply:
Indeed, the installation of LSP via mason has been removed per the request of gleam's team; see https://github.com/mason-org/mason-registry/pull/3872